curriculum/challenges/english/03-front-end-libraries/sass/apply-a-style-until-a-condition-is-met-with-while.english.md

```diff
@@ -91,7 +91,30 @@ tests:
 ## Solution
 <section id='solution'>
 
+To use the while loop within SASS a boolean condition has to setup. For this challenge we can take advantage of a counter and use that as part of the loop. The counter ```x``` is set to 1 before the ```@while``` statement. The boolean condition we're checking is ```$x < 11``` (is the variable x less than 11). If the value of $x is less than 11 it will continue to loop. 
```

Comment:
Hi @paulywill 

I think these comments will not be rendered in the UI. The solution is there to test against the tests in our CI. Meaning these are required for us to know that future edits to the challenge satisfy the solution.

You should remove them.

Reply:
@raisedadead, removing the comments. Should I keep the solution as is?